vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -143,7 +146,7 @@ class OnboardingViewModel @AssistedInject constructor(
     }
 
     private val matrixOrgUrl = stringProvider.getString(R.string.matrix_org_server_url).ensureTrailingSlash()
-    private val defaultHomeserverUrl = matrixOrgUrl
+    private val defaultHomeserverUrl = mdmService.getData(MdmData.DefaultHomeserverUrl, matrixOrgUrl)
```

Comment:
@bmarty don't you think we should move this mdmService call where the `defaultHomeserverUrl` is used?

Reply:
It should not change during the lifetime of the `OnBoardingViewModel`, but I prefer to avoid this scenario to occur. So better to read it once, and then it cannot change during the lifetime of the `OnBoardingViewModel`.